### PR TITLE
Improve documentation a bit

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: docs/src/other.md [hunk 1]
```diff
@@ -23,21 +23,25 @@ using Julia syntax features.
...
+- Call functions with global options via [`call_gap_func`](@ref)
+  or using Julia's keyword argument syntax. For example,
+  `Cyc(1.41421356 : bits:=20)` in GAP translates to
+  `GAP.Globals.Cyc(GAP.Obj(1.41421356); bits=20)` in Julia.
+
+- Access list/matrix entries via [`getindex`](@ref) and [`setindex!`](@ref)
+  respectively the corresponding Julia syntax (described there).
 
-- Access record components via [`getproperty`](@ref).
+- Access record components via [`getproperty`](@ref) and [`!etproperty!`](@ref)
```
Comment (ThomasBreuer) [inline docs/src/other.md:34]: setproperty!

**Review by ThomasBreuer — CHANGES_REQUESTED** [review]
just a typo in a cross-reference

File: docs/src/other.md (hunk 1 above)
Comment (fingolfin) [inline docs/src/other.md:34]:
```suggestion
- Access record components via [`getproperty`](@ref) and [`setproperty!`](@ref)
```

**Comment by fingolfin** [comment]
Fixed

**Review by ThomasBreuer — APPROVED** [review]

**Pull request merged**